rviz_default_plugins/src/rviz_default_plugins/displays/wrench/wrench_display.cpp

```diff
@@ -127,11 +133,24 @@ bool validateFloats(const geometry_msgs::msg::WrenchStamped & msg)
 
 void WrenchDisplay::processMessage(geometry_msgs::msg::WrenchStamped::ConstSharedPtr msg)
 {
-  if (!validateFloats(*msg)) {
-    setStatus(
-      rviz_common::properties::StatusProperty::Error, "Topic",
-      "Message contained invalid floating point values (nans or infs)");
-    return;
+  auto custom_msg = std::make_shared<geometry_msgs::msg::WrenchStamped>();
+  bool accept_NaN = accept_NaN_values_->getBool();
```

Comment:
This line should probably not be here but in the callback above where it is commented out...

```suggestion
```

Reply:
The 'accept_Nan' variable should be here to fetch the value. Based on this value, 'adjusted_msg' will be populated.